### ISSUE TITLE
Error in service BAT files (suggested fix included)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#154]
```
I believe files:

http://code.google.com/p/h2database/source/browse/trunk/h2/service/0_run_se
rver_debug.bat
http://code.google.com/p/h2database/source/browse/trunk/h2/service/1_instal
l_service.bat

that contain command (line 4):

copy /y /b ..\bin\h2-*.jar ..\bin\h2.jar

just try to get latest version of h2-*.jar (not concatenate all of them).

So it should look like:

for %i in (..\bin\h2-*.jar) do copy /y /b %i ..\bin\h2.jar

```

Original issue reported on code.google.com by `desummoner` on 22 Dec 2009 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I would like to keep the batch file as simple as possible.
Why do you have multiple h2-*.jar files in the same directory?
Maybe there is an easy way to fail if there are multiple h2-*.far file around.
```

Original comment by `thomas.t...@gmail.com` on 25 Dec 2009 at 4:24

**Comment by GoogleCodeExporter** [comment]
```
Not a problem really. The JAR is all we need.
=)

(I've got concatenated JAR file so just looked for cause. Windows installer 
ingnored an 
older version, so there was multiple h2-*.jar files.)
```

Original comment by `desummoner` on 25 Dec 2009 at 10:31

**Comment by GoogleCodeExporter** [comment]
```
I found a solution. I will fix it in the next release.
The Windows service wrapper will detect if multiple versions of H2 are 
installed.
```

Original comment by `thomas.t...@gmail.com` on 30 Jan 2010 at 3:16
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed with version 1.2.129.
```

Original comment by `thomas.t...@gmail.com` on 20 Feb 2010 at 9:43
- Changed state: **Fixed**
